src/index.tsx

```diff
@@ -271,6 +275,8 @@ export type ITimeUnit = 'seconds' | 'minutes' | 'hours';
 
 export type IMediaType = 'img' | 'video'
 
+export type IStatusDot = 'caution' | 'danger' | 'good' | 'neutral';
```

Comment:
We're going to need the blue color in here too which is `folder`. The problem (as I'm sure you're already thinking) is that we're using the status references from SMC and so that makes no sense here. 

I think perhaps we can add `highlight` as an option in the theme and here. Ideally we replace `folder` but only if we're certain that we're not using it in the code at all. Would you be able to add it to the theme manually? I've already added the reference to my theme generation setup so it shouldn't get lost later.

Reply:
Added the highlight as requested. 
How can you even think or removing folder?.
That color it's totally in fashion now days  🤭